### DISCUSSION TITLE
ON TRENDING

### DISCUSSION THREAD

**Discussion by TechStudent10** [ThioJoe/YT-Spammer-Purge#782] [Show and tell]
The repo is now on Trending on GitHub!

![image](https://user-images.githubusercontent.com/76978184/161628696-a0e37c5b-4224-4cb6-9f53-e03c6b5a59c6.png)

**Comment by KendallDoesCoding** [comment]
YES!! It's 11th on trending with 118 stars today, dang.

**Comment by ThioJoe** [comment]
Very epic

**Reply by TechStudent10** [reply]
Proud to be one of the people who "built it"

![image](https://user-images.githubusercontent.com/76978184/161864902-5fc842fa-2989-426e-a460-23b30ad12a05.png)

**Reply by Firecul** [reply]
Ditto
